### ISSUE TITLE
lintDirtyModulesOnly seems not working

### ISSUE THREAD

**Issue by coderars** [webpack-contrib/stylelint-webpack-plugin#167]
* Operating System: Windows 10 (1803)
* Node Version: 9.8.0
* NPM Version: 6.3.0
* webpack Version: 4.20.2
* stylelint-webpack-plugin Version: 0.10.5

This issue is for a:

- [x] **bug**
- [ ] **feature** request
- [ ] **modification** request

##### webpack.config.js

```js
      new StyleLintPlugin({
        context: path.resolve(paths.assets, 'sass'),
        // files: './resources/assets/sass/**/*.s?(a|c)ss',
        lintDirtyModulesOnly: true,
        syntax: "scss"
      })
```
### Expected Behavior

With `lintDirtyModulesOnly` set to `true`, after editing an `.scss` file (make some error) in watch or hot mode the plugin should check only that file and show the error in console. 
### Actual Behavior

The plugin is not checking the modified file (has no output in console). No matter if I use `context` or `files` option, it's the same. Without `lintDirtyModulesOnly` option everything works as excepted, I can see which separate `.scss` file(s) has error(s).

### Stackoverflow
Nothing found when searching for "lintDirtyModulesOnly"

Am I doing something wrong or misunderstood the purpose of this option?

**Comment by Nemo64** [comment]
I can confirm this. It seems to be impossible to get any errors with this option set to true.

**Comment by ricardogobbosouza** [comment]
Upgrade to latest version, if problem persists reopen this
